Dockerfile

```diff
@@ -60,6 +60,8 @@ RUN curl --location "${bazel_url}" > bazel.deb \
 # Install Node.js and npm.
 RUN curl --location https://deb.nodesource.com/setup_12.x | bash - \
   && apt-get install --no-install-recommends --yes nodejs \
+  && mkdir "/.npm" \
+  && chmod a+rwx "/.npm" \
```

Comment:
Problematic fave

Reply:
For background, NPM expects to be able to write to `~/.npm` during installs for caching. Since for our non root user this is the root dir, we have to create & enable rwx for this dir. 

For background see #1121